external-import/misp/src/misp.py

```diff
@@ -361,11 +357,11 @@ def run(self):
                 try:
                     events = self.misp.search("events", **kwargs)
                 except Exception as e:
-                    self.helper.log_error(str(e))
+                    self.helper.log_error(f"Error fetching misp event: {e}")
```

Comment:
I am not sure this change will work, you still need the string representation of the exception.
```suggestion
                    self.helper.log_error(f"Error fetching misp event: {str(e)}")
```

Reply:
It works, no need to cast to string two times. 

Example with exception on row 1149: 
```
{"timestamp": "2022-06-15T05:11:35.614126Z", "level": "ERROR", "name": "root", "message": "Error creating observable type Text with value https://www.virustotal.com/file/930939256e2c2fa30e7260897d96859c08cf767664e4bd3cedf156b6765b5413/analysis/1442496142/ (type=link): Unexpected properties for Text: (data)."}```